src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @typescript-eslint/indent */
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk, PayloadAction } from '@reduxjs/toolkit';
+import { Comment, CommentData } from '../../types/Comment';
+import * as commentsApi from '../../api/comments';
```

Comment:
Verify that the `commentsApi` functions (`getPostComments`, `createComment`, `deleteComment`) are correctly implemented and imported.